src/main/java/seedu/address/logic/parser/ImportExamCommandParser.java

```diff
@@ -0,0 +1,52 @@
+package seedu.address.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_IMPORT;
+
+import java.nio.file.Path;
+
+import seedu.address.logic.commands.ImportCommand;
+import seedu.address.logic.commands.ImportExamCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new {@code ImportExamCommand} object
+ */
+public class ImportExamCommandParser implements Parser<ImportExamCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the {@code ImportExamCommand}
+     * @param args the arguments to be parsed
+     * @return a new {@code ImportExamCommand} object
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public ImportExamCommand parse(String args) throws ParseException {
+        requireNonNull(args);
+        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_IMPORT);
+        if (!isPrefixPresent(
+                argMultimap)
+                || !argMultimap.getPreamble().isEmpty()) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, ImportExamCommand.MESSAGE_USAGE));
+        }
+
```

Comment:
!!! Should check if there are duplicate entries for "i/". Can do this with Argmultimap.verifyNoDuplicatePrefixesFor(PREFIX_IMPORT)

Reply:
Good catch!